filebeat/_meta/fields.common.yml

```diff
@@ -38,3 +38,9 @@
         Ingestion pipeline error message, added in case there are errors reported by
         the Ingest Node in Elasticsearch.
 
+    - name: beat.read_timestamp
```

Comment:
Not 100% sure about the location of this field as all the other fields in here are "beat" specific and rater constant. This one is event specific. But I also don't have a good alternative at the moment.